playbooks/openshift/deploy_aosi.yml

```diff
@@ -0,0 +1,18 @@
+# #### Dependencies
+#
+# - setup.yml
+# - repo_from_source.yml
+#
+# #### Install ose from built rpms
+# ## defined in install_ose.yml ##
+#
+# - drop yum repo config onto all nodes
+# - yum install packages
+# - configure and run atomic-openshift-installer
+
+---
+- hosts: all
+  sudo: '{{ sudo | default("no") }}'
+  user: '{{ user | default("root") }}'
```

Comment:
I think it is fine to have a default in the playbook so someone doesn't have to use one on the cli that is how we did it in the class